web_server/social_distribution/urls.py

```diff
@@ -18,7 +18,7 @@
 from . import views
 from users import views as users_views
 urlpatterns = [
-    path('', users_views.profile),
+    path('', views.main_page),
```

Comment:
This conflicts with the work that @phoebebai and @XuechunHou have done - the front page is the login page. This cannot be merged with this in place since it will override the login page.